### DISCUSSION TITLE
How to implement custom merge driver?

### DISCUSSION THREAD

**Discussion by shorton3** [gitpython-developers/GitPython#1430] [Q&A] [answered]
In our command line implementation, we have created a git custom merge driver by modifying .gitattributes and .gitconfig.  Is there a way to have the same custom merge driver used with gitpython?  Or can we implement something similar?  The custom merge driver automatically takes/uses the version with the latest commit time.  Thanks!

**Comment by stephan1373** [comment]
This was a user environment error for us - the merge driver had to be on the path (and we thought it was already).
Anybody who glanced at this - thank you for understanding, sorry for the question.

**Comment by Byron** [comment, marked as answer]
Merge drivers will only be picked up if merges are performed by git itself, which can typically be achieved by using `repo.git.merge(…)`. Using git directly is [described here in greater detail](https://gitpython.readthedocs.io/en/stable/tutorial.html#using-git-directly).
